src/main/scala/mesosphere/mesos/TaskGroupBuilder.scala

```diff
@@ -434,6 +433,46 @@ object TaskGroupBuilder {
     builder
   }
 
+  /**
+    * Computes all port env vars for the given container.
+    * Form: PORT{INDEX}=123
+    */
+  private[this] def containerPortEnvVars(
```

Comment:
This should only do named endpoints, e.g.

ENDPOINT_${NAME}_CONTAINER_PORT_${PORTNUM}=${HOSTPORT}